Source/cmMakefileExecutableTargetGenerator.cxx

```diff
@@ -631,7 +631,12 @@ void cmMakefileExecutableTargetGenerator::WriteExecutableRule(bool relink)
     this->LocalGenerator->SetLinkScriptShell(false);
   }
 
-  this->CreateLinkCommandFile(real_link_commands, dependsForJson,
+  std::vector<std::string> linkCommands = real_link_commands;
+  if (this->GeneratorTarget->GetPropertyAsBool("LINK_WHAT_YOU_USE")) {
```

Comment:
Where do you filter this one:
https://github.com/Software-Analysis-Team/link-commands
"command" : "/usr/local/bin/cmake -E rm -f CMakeFiles/library.dir/lib.a",